calculator/src/App.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import Intro from "./components/Intro";
+import Form from "./components/Form";
+import Chart from "./components/Chart";
+import "./App.css";
+
+function App() {
```

Comment:
You can set up a class component here and write state as one object - saves a lot of time :)